db/migrate/20201030102020_enlarge_report_status.rb

```diff
@@ -0,0 +1,29 @@
+class EnlargeReportStatus < ActiveRecord::Migration[6.0]
+  def up
+    remove_index :reports, :status
+
+    User.as_anonymous_admin do
+      User.without_auditing do
+        Report.unscoped.where(origin: "Puppet").find_each do |report|
+          old_calc = ConfigReportStatusCalculator.new(bit_field: report.read_attribute(:status), size: 6)
+          new_calc = ConfigReportStatusCalculator.new(counters: old_calc.status, size: 10)
+          report.update_column(:status, new_calc.calculate)
+        end
+      end
+    end
```

Comment:
The rest is iterating through reports and converting from 6 bit word size to 10 bits. And vice versa.